src/plot/series/mark-series.js

```diff
@@ -62,10 +62,11 @@ class MarkSeries extends AbstractSeries {
          ref="container"
          transform={`translate(${marginLeft},${marginTop})`}>
         {data.map((d, i) => {
+          const size = sizeFunctor ? sizeFunctor(d) : DEFAULT_SIZE;
+          const x = xFunctor(d);
```

Comment:
on going nit: we have been using the word functor incorrectly in react-vis (it is not a function-object), and so should seek to just call it the variable, eg x instead of xFunctor